Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    patch_ruby (1.3.0)
+    patch_ruby (1.3.1)
```

Comment:
Is there a PR in the code gen repo as well? The versions can be set there in the config files for each language.